sale_restrict/models/sale_order.py

```diff
@@ -0,0 +1,24 @@
+# See LICENSE file for full copyright and licensing details.
```

Comment:
@PratikVaza-SerpentCS did we make sure we need this in v16?  v16 mate navu feature add karyu koi?  we shud.
aap free na ho to  I can add @pragatipatel-serpentcs to lend a hand of support. She can spend 30 mins next week for this.

Reply:
> @PratikVaza-SerpentCS did we make sure we need this in v16? v16 mate navu feature add karyu koi? we shud. aap free na ho to I can add @pragatipatel-serpentcs to lend a hand of support. She can spend 30 mins next week for this.

Sale restrict module adds constraints or another checking layer when confirming sales order for products whose unit price is zero. It will restrict the user to add zero amount in product.

As a suggestion, we can give a option like whether to check the lines when confirming the sales order on user level or company level.

Can you please add @pragatipatel-serpentcs? As I am busy with live task.